lib/foreman_remote_execution/engine.rb

```diff
@@ -155,6 +155,11 @@ class Engine < ::Rails::Engine
               default: 'Job - Invocation Report',
               full_name: N_('Job Invocation Report Template'),
               collection: proc { ForemanRemoteExecution.job_invocation_report_templates_select }
+            setting 'remote_execution_time_to_pickup',
+              type: :integer,
+              description: N_('Time in seconds within which the host has to pick up a job. If the job is not picked up within this limit, the job will be cancelled.'),
```

Comment:
Should we mention that by default it's 24 hours?

Reply:
Done